components/Typography/tests/unit/UIFont+MaterialTypographyTests.m

```diff
@@ -42,4 +42,36 @@ - (void)testMDC_standardFontForMaterialTextStyleReturnsEquivalentFonts {
   }
 }
 
+- (void)testScaledFontForTraitEnvironmentOniOS10AndAboveConsultsTheTraitEnvironment {
+  if (@available(iOS 10.0, *)) {
+    // Given
+    UIFont *font = [UIFont systemFontOfSize:22.0];
+    font =
+        [[MDCFontScaler scalerForMaterialTextStyle:MDCTextStyleHeadline1] scaledFontWithFont:font];
+    MockTraitEnvironment *traitEnvironment = [[MockTraitEnvironment alloc] init];
+    traitEnvironment.traitCollection = [UITraitCollection
+        traitCollectionWithPreferredContentSizeCategory:UIContentSizeCategoryExtraLarge];
+
+    // When
+    UIFont *scaledFont = [font mdc_scaledFontForTraitEnvironment:traitEnvironment];
+
+    // Then
+    XCTAssertGreaterThan(scaledFont.pointSize, font.pointSize);
+  }
+}
+
+- (void)testScaledFontForTraitEnvironmentWithNoApplicationOniOS9DoesNothing {
```

Comment:
I'm a little confused.  How do you know or guarantee tht this is on iOS 9 and that there is no `UIApplication.sharedApplication`?

Reply:
Ah good catch! Will update.

Reply:
How do you know or guarantee there is no `UIApplication.sharedApplication`?

Reply:
We assert that there is not one in the test.